### ISSUE TITLE
[Feature Request] Support for jsonpath-plus

### ISSUE THREAD

**Issue by TheBay0r** [getinsomnia/insomnia#1964]
Hello everybody,

at the moment I have the issue that one json returned contains an array of objects that have a uuid as their key. And for a subsequent request I would need to grab this uuid of one of the objects contained in the array. 

Example data structure:
```json
{
  "myarray": [
    {
      "7d27fbf6-df86-48cf-99e7-ad0e9ae3bc68": {
        "foo": "bar",
        "more": "data"
      }
    }
  ]
}
```
At the moment it is not possble - at least i couldn't find it - to grab the key(s) of an object with the standard jsonpath implementation.

Would it be an option to add support for [jsonpath-plus](https://www.npmjs.com/package/jsonpath-plus)? This package supports the `~`-operator to grab the key of an object. Therefore the request for my example data structure would look like this: `$.myarray[0].*~`

I'm not aware of any alternative solution but would be more than happy to have suggestions.

**Comment by TheBay0r** [comment]
Okay, I was looking a little bit more into the issue and even tried to update the code. 
I now found a little bigger problem (which could be solved, but so far it wouldn't look nice). If an object has multiple keys, this is returned as multiple results and therefore would trigger the error: 
`Returned more than one result: …`

In my example I disabled that check and return just a stringified json. Which helps getting results, but probably isn't what a user would want. My example right now looks like this:
I'm parsing the JSON response for [this request](http://dummy.restapiexample.com/api/v1/employees).
<img width="480" alt="Screenshot 2020-02-25 at 14 15 02" src="https://user-images.githubusercontent.com/8627051/75251062-23015b80-57da-11ea-97a1-b1bbef0d42d1.png">
This would look somewhat okay-ish, but probably doesn't fulfil a purpose…


Now jsonpath-plus would even allow me something really nasty:
<img width="480" alt="Screenshot 2020-02-25 at 14 22 59" src="https://user-images.githubusercontent.com/8627051/75251166-5a700800-57da-11ea-8b07-a425445c3c41.png">
1. probably a usecase that should not be supported? 
2. it makes the problem of my quickfix visible, that I just return a stringified array
